db/migrations/versions/016_rename_event_data_blob.py

```diff
@@ -0,0 +1,117 @@
+"""empty message
+
+Revision ID: 016_rename_event_data_blob
+Revises: 015_add_jsonb_to_submission
+Create Date: 2024-03-05 14:35:05.518333
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "016_rename_event_data_blob"
+down_revision = "015_add_jsonb_to_submission"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("funding", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("data_blob", postgresql.JSONB(astext_type=sa.Text()), nullable=True))
+        batch_op.drop_column("event_data_blob")
```

Comment:
Would this result in us losing whatever data was in `event_data_blob`? Does it need a step in between to set the contents of `data_blob` to be the same as `event_data_blob` before it's dropped, similar to what you did when setting up the `event_data_blob` columns originally?

Reply:
Yes, it should be just renamed